phpBB/styles/prosilver/template/navbar_header.html

```diff
@@ -72,12 +72,12 @@
 		</li>
 		<!-- IF S_DISPLAY_PM -->
 			<li class="small-icon icon-pm rightside" data-skip-responsive="true">
-				<a href="{U_PRIVATEMSGS}" role="menuitem"><span>{L_PRIVATE_MESSAGES} [</span><strong>{PRIVATE_MESSAGE_COUNT}</strong><span>]</span></a>
+				<a href="{U_PRIVATEMSGS}" role="menuitem"><span>{L_PRIVATE_MESSAGES} </span><!-- IF PRIVATE_MESSAGE_COUNT --><strong class="notification-alert">{PRIVATE_MESSAGE_COUNT}</strong><!-- ENDIF --><span></span></a>
```

Comment:
why is any js needed if there is no count by these lines it should not render? Just curious